src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball firstBall = new Lottery().getRandomBall();
+        Ball secondBall = new Lottery().getRandomBall();
+        Ball thirdBall = new Lottery().getRandomBall();
+
+        System.out.println(firstBall.toString());
+        System.out.println(secondBall.toString());
+        System.out.println(thirdBall.toString());
```

Comment:
DRY - dont repeat yourself. code duplicating is a bad practice and can cause bugs. use loop instead